pyscf/md/integrators.py

```diff
@@ -310,6 +341,11 @@ def dump_input(self, verbose=None):
         log.info('******** BOMD flags ********')
         log.info('dt = %f', self.dt)
         log.info('Iterations = %d', self.steps)
+        
+        if self.T is not None:
```

Comment:
Similar to `check_sanity`, perhaps have a overridden method in the subclass like:
```python
def dump_input(self, verbose=None):
    log = logger.new_logger(self, verbose)
    self.super().dump_input(verbose=log)
    log.info('Temperature = %f', self.T)
    log.info('tau_T = %f', self.taut)
```

Reply:
Done!